webpack.config.js

```diff
@@ -10,6 +10,10 @@ module.exports = {
         test: /\.tsx?$/,
         use: "ts-loader",
         exclude: /node_modules/
+      },
+      {
+        test: /\.(yaml|md)?$/,
+        loader: "url-loader?limit=100000"
```

Comment:
can we use the object format for configuring the loader?

ie: from the `url-loader` docs: https://www.npmjs.com/package/url-loader

```
        use: [
          {
            loader: 'url-loader',
            options: {
              limit: 8192,
            },
          },
```

Reply:
let me test it out

Reply:
works. thanks